bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCgnirs.html

```diff
@@ -486,60 +428,57 @@
 	<p>
 	&nbsp;
 	</p>
-	<!-- Observing conditions definition-->
 
+	<!-- Observing conditions definition-->
 	<p>
 	<span style="color: #ff0000"><b>Observing condition constraints</b></span>
```

Comment:
Just a question: given that we're moving towards capitalizing things like Image Quality, should this also be capitalized as `Observing Condition Constraints`?

Reply:
I'm on the fence.  I have capitalized "Image Quality", "Cloud Cover", and "Water Vapor" because they are frequently abbreviated as "IQ","CC", and "WV".